card/mod/05_standard/set/type/search_type.rb

```diff
@@ -217,7 +216,7 @@ def default_search_params
     total = card.count search_params
     return '' if limit >= total # should only happen if limit exactly equals the total
 
-    @paging_path_args = { :limit => limit, :item=> inclusion_defaults[:view] }
+    @paging_path_args = { :limit => limit, :item=> inclusion_defaults(card)[:view] }
```

Comment:
good point.  in pointers and searches we probably don't want anything card-specific